<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Live Streaming with a Webcam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbSevSULogo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>293</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/png/sevsu.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbCopyright">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>750</y>
      <width>202</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>© 2023 СевГУ. Все права защищены.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCamera">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>CAMERA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edIPCamera">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>http://localhost:5000/video_feed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>570</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>УСТАНОВИТЬ СОЕДИНЕНИЕ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCopyright_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>640</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Live Streaming with a Webcam</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>lblCamera</zorder>
   <zorder>lbSevSULogo</zorder>
   <zorder>lbCopyright</zorder>
   <zorder>edIPCamera</zorder>
   <zorder>btnConnect</zorder>
   <zorder>lbCopyright_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
